ci-operator/config/openshift/external-secrets-operator-release/openshift-external-secrets-operator-release-release-0.14.yaml

```diff
@@ -0,0 +1,40 @@
+base_images:
+  base:
+    name: "4.19"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.23
+promotion:
+  to:
+  - name: "4.19"
+    namespace: ocp
+releases:
+  initial:
+    integration:
+      name: "4.19"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.19"
+      namespace: ocp
```

Comment:
similarly, as we're not building any images that would contribute to core openshift payload
my understanding would be that we can directly use either:

Stable:
```
releases:
  latest:
    release:
      // ...
```

Or, nightly:
```
releases:
  latest:
    candidate:
      // ...
```